### DISCUSSION TITLE
showopenfilepicker is not defined

### DISCUSSION THREAD

**Discussion by hupo4321** [lichtblick-suite/lichtblick#236] [General]
When I access the started server through Google Chrome web on another computer and want to load a layouts file, it prompts Referenceerror: showopenfilepicker is not defined, as shown below
![image-2024-10-22_12-54-33](https://github.com/user-attachments/assets/01c5f6dc-a9cb-4468-ae6e-2b3bea9abc74)

**Comment by aneuwald-ctw** [comment]
Hello @hupo4321, sorry for the late answer!

This bug should be fixed by now, please check the latest version of Lichtblick on  https://github.com/lichtblick-suite/lichtblick/releases
